samples/net/slip_throughput_client/boards/native_sim.overlay

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
About the samples, could you please place them in a separate commit. Also we could create `samples/net/slip` and place the related samples there.